olive/passes/onnx/conversion.py

```diff
@@ -420,34 +421,41 @@ def _convert_distributed_model_on_device(
         torch_dtype: Optional[torch.dtype] = None,
     ) -> DistributedOnnxModel:
         """Convert a DistributedPyTorchModel to a DistributedOnnxModel."""
-        from mpi4py.futures import MPIPoolExecutor
-
         pass_config = config
         model_config = model.to_json()["config"]
         world_size = model.num_ranks
         output_model_path = str(Path(output_model_path).with_suffix(""))
+        use_gpu = self.accelerator_spec.accelerator_type == Device.GPU
```

Comment:
should use_gpu just check `device`? We don't want the conversion pass to depend on the target accelerator type but pass `device` if needed.
device is cpu by default for regular conversion and cuda for distributed.

Reply:
Done!